pkg/controller/volume/attach_detach_controller.go

```diff
@@ -261,7 +261,7 @@ func (adc *attachDetachController) processPodVolumes(
 
 	// Process volume spec for each volume defined in pod
```

Comment:
We need to fix the controller-manager manifest to include /usr/libexec/kubernetes hostpath volume